projects/extension/ai/vectorizer.py

```diff
@@ -8,7 +8,7 @@
 from .utils import get_guc_value
 
 GUC_VECTORIZER_URL = "ai.external_functions_executor_url"
-DEFAULT_VECTORIZER_URL = "http://localhost:8000"
+DEFAULT_VECTORIZER_URL = "http://0.0.0.0:8000"
```

Comment:
This change is fixing CI errors connecting to the test server.

Reply:
nice catch!